src/style.scss

```diff
@@ -26,4 +26,18 @@ h1{
  }
   #a5{
       color:green
+  }
+
+  .list{
+    text-align: center;
+    padding-block-start: 20px;

```

Comment:
never add px in css .try achieving the same through rem.
It help is cross device development.